apps/challenges/urls.py

```diff
@@ -23,8 +23,6 @@
         views.get_challenge_by_pk, name='get_challenge_by_pk'),
     url(r'challenge$', views.get_challenges_based_on_teams,
         name='get_challenges_based_on_teams'),
-    url(r'(?P<challenge_pk>[0-9]+)/challenge_phase_split$', views.challenge_phase_split_list,
```

Comment:
@S2606 We don't have to remove the endpoint. We need to remove only the pagination from the API.
So, please revert these changes.